cfme/tests/intelligence/reports/test_import_export_reports_widgets.py

```diff
@@ -182,9 +182,9 @@ def test_reports_invalid_file(appliance, yaml_name):
             2. Import `invalid_yaml` yaml that has no yaml data.
     """
     if yaml_name == "invalid_yaml":
-        message = "Error during 'upload': undefined method `keys' for \"i\":String"
+        message = r".*Error during .*upload.*: undefined method `keys.* for .*i.*:String.*"
     else:
```

Comment:
Are you sure about spaces between words?
```message = r".*Error during .*upload.*: undefined method `keys.* for .*i.*:String.*"```

Can we write like this - ```message = r".*Error during.*upload.*: undefined method `keys.*for.*i.*:String.*"```

Reply:
Yeah, I think it will be better to leave the space out. Thanks!

Reply:
What are we trying to fix here? using `.*` is making this expression very greedy, if there's an issue with the number of whitespace characters in the string in the UI, we should use `\s+` to express "one or more whitespace characters"